test/stack_test.c

```diff
@@ -45,6 +45,8 @@ DEFINE_SPECIAL_STACK_OF_CONST(uchar, unsigned char)
 DEFINE_STACK_OF(SS)
 DEFINE_STACK_OF_CONST(SU)
 
+IMPLEMENT_COMPARISONS(char, char, "%c")
```

Comment:
I'd like to see a static variant of `IMPLEMENT_COMPARISONS`...  it would be fitting here.